align_system/utils/alignment_utils.py

```diff
@@ -169,21 +173,21 @@ def __call__(self, kdma_values, target_kdmas, misaligned=False, kde_norm='global
         likelihoods = []
         choices = list(kdma_values.keys())
         for choice in choices:
-            distance = 0.
+            total_likelihood = 0.
             for target_kdma in target_kdmas:
                 if isinstance(target_kdma, KDMAValue):
                     target_kdma = target_kdma.to_dict()
 
                 target_kde = kde_utils.load_kde(target_kdma, kde_norm)
                 predicted_samples = kdma_values[choice][target_kdma.kdma]
                 log_likelihoods = target_kde.score_samples(np.array(predicted_samples).reshape(-1, 1))
-                total_likelihood = np.sum(np.exp(log_likelihoods))
+                total_likelihood += np.sum(np.exp(log_likelihoods))
```

Comment:
@jadie1 I would especially like you to review that this particular change isn't wrong. I *think* I caught a bug, but may be misunderstanding. Wouldn't have affected our current results since it's only in play with multi KDMAs

Reply:
@eveenhuis I think you did catch a bug. This fix looks correct, we want to total likelihood of each choice to be the sum across all target KDMAs